### PR TITLE
Release v3.0.1

### PR DESCRIPTION
**Pull request by EdwinGuzman**
### Adds

- Adds the `"editorMode"` option (pencil) to the `Icon` component.
- Updates the `ul` bullet color in the `List` component.
- Adds the `Errors` page to the `Accessibility Guide` section of Storybook.

### Updates

- Updates default heading size to `heading4` on `Modal` component.
- Updates mobile styles for the `SocialMediaLinks` component

### CODE REVIEW

File: src/components/Modal/Modal.mdx [hunk 1]
```diff
@@ -222,10 +223,16 @@ export const ModalStory = (args) => {
 
 <ArgTypes of={ModalStories.useModalStory} />
 
+## Default Heading Text
+
+By default, the `Modal` component will render a DS level `h2` and size `h4` `Heading` component when
```
Comment (jackiequach) [inline src/components/Modal/Modal.mdx]: This should be size `heading4` instead of `h4`

File: src/components/Icons/iconChangelogData.ts [hunk 2]
```diff
@@ -9,6 +9,13 @@
 import { ChangelogData } from "../../utils/ComponentChangelogTable";
 
 export const changelogData: ChangelogData[] = [
+  {
+    date: "2024-03-28",
+    version: "3.0.1",
+    type: "Update",
+    affects: ["Documentation", "Functionality"],
+    notes: ["Addsed the editorMode (pencil) icon."],
```
Comment (oliviawongnyc) [inline src/components/Icons/iconChangelogData.ts]: Not too important, but typo in "Added"

**Review by jackiequach — APPROVED** [review]
Looks good except for minor typo

**Review by oliviawongnyc — APPROVED** [review]
Looks good to me! Pointed out one typo.

**Review by 7emansell — APPROVED** [review]
Lgtm except typos which were my fault 😬

**Pull request merged**